rio/src/RIO/Prelude/RIO.hs

```diff
@@ -1,11 +1,28 @@
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE FlexibleInstances #-}
 {-# LANGUAGE GeneralizedNewtypeDeriving #-}
 {-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE FunctionalDependencies #-}
+{-# LANGUAGE UndecidableInstances #-}
 module RIO.Prelude.RIO
   ( RIO (..)
   , runRIO
   , liftRIO
+  -- * SomeRef for Writer/State interfaces
+  , SomeRef
+  , HasStateRef
```

Comment:
Both this and the following need to be exported with the member functions, e.g. `HasStateRef (..)`.

Reply:
done